src/languages/javascript.coffee

```diff
@@ -15,14 +15,17 @@ module.exports = {
   Supported Grammars
   ###
   grammars: [
-    "JavaScript"
+    "JavaScript",
+    "JavaScript for Automation (JXA)"
   ]
 
   ###
   Supported extensions
   ###
   extensions: [
-    "js"
+    "js",
+    "scpt",
+    "applescript"
```

Comment:
Thanks – I have now added the full list.

	'scpt'
	'jxa'
	'jxainc'
	'applescript'